# encoding: utf-8
# module h5py._errors
# from C:\Users\Doly\Anaconda3\lib\site-packages\h5py\_errors.cp37-win_amd64.pyd
# by generator 1.147
# no doc

# imports
import builtins as __builtins__ # <module 'builtins' (built-in)>

# functions

def filename_decode(filename): # reliably restored by inspect
    """
    Decode filename used by HDF5 library.
    
        Due to how HDF5 handles filenames on different systems, this should be
        called on any filenames passed from the HDF5 library. See the documentation
        on filenames in h5py for more information.
    """
    pass

def filename_encode(filename): # reliably restored by inspect
    """
    Encode filename for use in the HDF5 library.
    
        Due to how HDF5 handles filenames on different systems, this should be
        called on any filenames passed to the HDF5 library. See the documentation on
        filenames in h5py for more information.
    """
    pass

def silence_errors(*args, **kwargs): # real signature unknown
    """ Disable HDF5's automatic error printing in this thread """
    pass

def unsilence_errors(*args, **kwargs): # real signature unknown
    """ Re-enable HDF5's automatic error printing in this thread """
    pass

# no classes
# variables with complex values

_exact_table = {
    (
        864691128455135233,
        864691128455135396,
    ): 
        ValueError
    ,
    (
        864691128455135235,
        864691128455135289,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
    (
        864691128455135238,
        864691128455135308,
    ): 
        OSError
    ,
    (
        864691128455135240,
        864691128455135289,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
    (
        864691128455135242,
        864691128455135289,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
    (
        864691128455135252,
        864691128455135289,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
    (
        864691128455135256,
        864691128455135289,
    ): 
        TypeError
    ,
    (
        864691128455135261,
        864691128455135289,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
    (
        864691128455135261,
        864691128455135382,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
    (
        864691128455135266,
        864691128455135384,
    ): 
        '<value is a self-reference, replaced by this string>'
    ,
}

_minor_table = {
    864691128455135267: OSError,
    864691128455135268: '<value is a self-reference, replaced by this string>',
    864691128455135269: '<value is a self-reference, replaced by this string>',
    864691128455135270: '<value is a self-reference, replaced by this string>',
    864691128455135271: '<value is a self-reference, replaced by this string>',
    864691128455135272: '<value is a self-reference, replaced by this string>',
    864691128455135277: ValueError,
    864691128455135303: KeyError,
    864691128455135309: '<value is a self-reference, replaced by this string>',
    864691128455135310: '<value is a self-reference, replaced by this string>',
    864691128455135311: '<value is a self-reference, replaced by this string>',
    864691128455135312: '<value is a self-reference, replaced by this string>',
    864691128455135313: '<value is a self-reference, replaced by this string>',
    864691128455135315: '<value is a self-reference, replaced by this string>',
    864691128455135321: '<value is a self-reference, replaced by this string>',
    864691128455135322: '<value is a self-reference, replaced by this string>',
    864691128455135323: '<value is a self-reference, replaced by this string>',
    864691128455135324: '<value is a self-reference, replaced by this string>',
    864691128455135325: '<value is a self-reference, replaced by this string>',
    864691128455135326: '<value is a self-reference, replaced by this string>',
    864691128455135327: '<value is a self-reference, replaced by this string>',
    864691128455135328: '<value is a self-reference, replaced by this string>',
    864691128455135329: '<value is a self-reference, replaced by this string>',
    864691128455135330: '<value is a self-reference, replaced by this string>',
    864691128455135331: '<value is a self-reference, replaced by this string>',
    864691128455135366: '<value is a self-reference, replaced by this string>',
    864691128455135377: '<value is a self-reference, replaced by this string>',
    864691128455135380: '<value is a self-reference, replaced by this string>',
    864691128455135381: NotImplementedError,
    864691128455135382: TypeError,
    864691128455135383: '<value is a self-reference, replaced by this string>',
    864691128455135384: '<value is a self-reference, replaced by this string>',
    864691128455135385: '<value is a self-reference, replaced by this string>',
    864691128455135386: '<value is a self-reference, replaced by this string>',
    864691128455135392: '<value is a self-reference, replaced by this string>',
    864691128455135396: '<value is a self-reference, replaced by this string>',
}

__loader__ = None # (!) real value is '<_frozen_importlib_external.ExtensionFileLoader object at 0x000001E82A23ADA0>'

__pyx_capi__ = {
    'set_error_handler': None, # (!) real value is '<capsule object "__pyx_t_4h5py_7_errors_err_cookie (__pyx_t_4h5py_7_errors_err_cookie)" at 0x000001E82A209690>'
    'set_exception': None, # (!) real value is '<capsule object "int (void)" at 0x000001E82A209660>'
}

__spec__ = None # (!) real value is "ModuleSpec(name='h5py._errors', loader=<_frozen_importlib_external.ExtensionFileLoader object at 0x000001E82A23ADA0>, origin='C:\\\\Users\\\\Doly\\\\Anaconda3\\\\lib\\\\site-packages\\\\h5py\\\\_errors.cp37-win_amd64.pyd')"

__test__ = {}

